File: mpp/lab3 2/lab3/src/main/java/org/example/domain/Spectacol.java
package org.example.domain;

import java.time.LocalDate;

public class Spectacol extends Entity<Long> {
    private LocalDate data;
    private String loc;
    private int numarLocDisponibile;
    private int numarLocVandute;
    private String numeArtist;

    public Spectacol(Long id, LocalDate data, String loc, int numarLocDisponibile, int numarLocVandute, String numeArtist) {
        super(id);
        this.data = data;
        this.loc = loc;
        this.numarLocDisponibile = numarLocDisponibile;
        this.numarLocVandute = numarLocVandute;
        this.numeArtist = numeArtist;
    }

    public LocalDate getData() {
        return data;
    }

    public void setData(LocalDate data) {
        this.data = data;
    }

    public String getLoc() {
        return loc;
    }

    public void setLoc(String loc) {
        this.loc = loc;
    }

    public int getNumarLocDisponibile() {
        return numarLocDisponibile;
    }

    public void setNumarLocDisponibile(int numarLocDisponibile) {
        this.numarLocDisponibile = numarLocDisponibile;
    }

    public int getNumarLocVandute() {
        return numarLocVandute;
    }

    public void setNumarLocVandute(int numarLocVandute) {
        this.numarLocVandute = numarLocVandute;
    }

    public String getNumeArtist() {
        return numeArtist;
    }

    public void setNumeArtist(String numeArtist) {
        this.numeArtist = numeArtist;
    }
}

File: mpp/Spectacole/domain/Spectacol.cs
using System;

namespace lab_mpp3.domain
{
    public class Spectacol : Entity<long>
    {
        private DateTime data;
        private string loc;
        private int numarLocDisponibile;
        private int numarLocVandute;
        private string numeArtist;

        public Spectacol()
        {
            data = DateTime.MinValue;
            loc = "";
            numarLocDisponibile = 0;
            numarLocVandute = 0;
            numeArtist = "";
        }

        public Spectacol(DateTime data, string loc, int numarLocDisponibile, int numarLocVandute, string numeArtist)
        {
            this.data = data;
            this.loc = loc;
            this.numarLocDisponibile = numarLocDisponibile;
            this.numarLocVandute = numarLocVandute;
            this.numeArtist = numeArtist;
        }

        public DateTime Data
        {
            get { return data; }
            set { data = value; }
        }

        public string Loc
        {
            get { return loc; }
            set { loc = value; }
        }

        public int NumarLocDisponibile
        {
            get { return numarLocDisponibile; }
            set { numarLocDisponibile = value; }
        }

        public int NumarLocVandute
        {
            get { return numarLocVandute; }
            set { numarLocVandute = value; }
        }

        public string NumeArtist
        {
            get { return numeArtist; }
            set { numeArtist = value; }
        }

        public override string ToString()
        {
            return $"Spectacol: {numeArtist}, Data: {data}, Loc: {loc}, Numar Locuri Disponibile: {numarLocDisponibile}, Numar Locuri Vandute: {numarLocVandute}";
        }
    }
}